### DISCUSSION TITLE
STM32CUBEMX and CUBEIDE

### DISCUSSION THREAD

**Discussion by JingtaiChen** [stm32duino/Arduino_Core_STM32#2084] [General]
How to use cubemx and arduino ide for joint debugging?

**Comment by fpistm** [comment]
Hi @CJT1111 
You can use the Arduino IDE 2.x to debug now.
Anyway, you can use STM32Cube IDE since the version 1.13.0 as it supports CMake import.
Like the Core also provide CMake support you can create your project then import it in CubeIDE.
I have to uupdate the wiki to explain the step but don't know when.

**Comment by JingtaiChen** [comment]
Thank you
